catalogs/catalog-hive/src/main/java/org/apache/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -235,7 +234,6 @@ private void initKerberosIfNecessary(Map<String, String> conf, Configuration had
 
         LOG.info("krb5 path: {}", System.getProperty("java.security.krb5.conf"));
         refreshKerberosConfig();
-        KerberosName.resetDefaultRealm();
```

Comment:
` KerberosName.resetDefaultRealm();` has no real effect in the current context, we can freely remove it.